src/core/lib/transport/promise_endpoint.cc

```diff
@@ -88,18 +85,17 @@ void PromiseEndpoint::ReadCallback(
     if (read_buffer_.Length() < num_bytes_requested) {
       // A further read is needed.
       // Set read args with number of bytes needed as hint.
-      requested_read_args = {
-          static_cast<int64_t>(num_bytes_requested - read_buffer_.Length())};
+      grpc_event_engine::experimental::EventEngine::Endpoint::ReadArgs
+          read_args;
+      read_args.read_hint_bytes =
```

Comment:
Minor: Could it initialize in the same line?

Reply:
Done. Thanks for suggestion.